app/views/react.scala.html

```diff
@@ -1,8 +1,18 @@
 @import assets.AssetsResolver
-@(title: String, id: String, js: String, description: Option[String] = None)(implicit assets: AssetsResolver, request: RequestHeader)
+@(
+    title: String,
+    id: String,
+    js: String,
+    contributionsPayPalEndpoint: String,
```

Comment:
The `react.scala.html` template is a generic template for pages that run react. Since it is generic, I would not add the contributionPayPalEndpoint to it. I would create a new template for the bundleLandingPage or a generic PayPalTemplate.